### PR TITLE
Add LED's to display robot state

### LINKED ISSUE #40: Status LEDs
Should be used to display the status of the robot in a simple way.

### PR DESCRIPTION
**Pull request by SturrockPeter**
Finished LED integration within arm states

Resolves #40

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Arm.java [hunk 1]
```diff
@@ -129,18 +132,22 @@ public Command goToSetpoint(Setpoint setpoint) {
...
 
-      case kSpeaker:
+      case kIntake:
         position = 3.7;
+        lEDPwm.setSpeed(0.57);
         break;
 
       case kStowed:
         position = 3.7;
+        lEDPwm.setSpeed(0.99);
```
Comment (spacey-sooty) [inline src/main/java/frc/robot/subsystems/Arm.java:150]: Could we have comments to say what that actually displays, its not obvious from the code

File: src/main/java/frc/robot/subsystems/Arm.java [hunk 2]
```diff
@@ -42,6 +43,8 @@ public enum Setpoint {
   private DoubleLogEntry log_ff_output = new DoubleLogEntry(m_log, "/arm/ff/output");
   private StringLogEntry log_setpoint = new StringLogEntry(m_log, "/arm/setpoint");
 
+  PWM lEDPwm = new PWM(0);
```
Comment (spacey-sooty) [inline src/main/java/frc/robot/subsystems/Arm.java:46]: this port needs to be determined still no?
```suggestion
  private PWM m_led = new PWM(0);
```

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
IMO we should display info about intaking and robot position as well.
Needs coordination with driveteam and @totallysomeoneyoudontknow
cc @Hazzer890 I'm sure you have an opinion on what the LEDs should do

**Comment by SturrockPeter** [comment]
> IMO we should display info about intaking and robot position as well. Needs coordination with driveteam and @totallysomeoneyoudontknow cc @Hazzer890 I'm sure you have an opinion on what the LEDs should do

currently working off LED patterns decided with Layla during season.

File: src/main/java/frc/robot/subsystems/Arm.java (hunk 2 above)
Comment (SturrockPeter) [reply src/main/java/frc/robot/subsystems/Arm.java:46]: > this port needs to be determined still no?

maybe, was going off memory, could also be port 1

File: src/main/java/frc/robot/subsystems/Arm.java [hunk 3]
```diff
@@ -129,18 +132,22 @@ public Command goToSetpoint(Setpoint setpoint) {
...
 
-      case kSpeaker:
+      case kIntake:
         position = 3.7;
+        lEDPwm.setSpeed(0.57); // hot pink //
         break;
 
       case kStowed:
         position = 3.7;
+        lEDPwm.setSpeed(0.99); // blank //
```
Comment (spacey-sooty) [inline src/main/java/frc/robot/subsystems/Arm.java:150]: Remove trailing slashes

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

**Pull request closed without merging**